test/unittests/instantiate_test.cpp

```diff
@@ -14,12 +14,12 @@ using namespace fizzy::test;
 
 namespace
 {
-uint64_t call_table_func(Instance& instance, size_t idx)
+uint32_t call_table_func(Instance& instance, size_t idx)
 {
     const auto& elem = (*instance.table)[idx];
     const auto res = elem->function(instance, {}, 0);
     EXPECT_TRUE(res.has_value);
-    return res.value;
+    return res.value.as<uint32_t>();
```

Comment:
Here, too: if table function would return incorrect `i64` value, we could miss it, if we check only lower 32 bit.